packages/schema/src/res/stdlib.zmodel

```diff
@@ -376,6 +376,11 @@ attribute @@deny(_ operation: String, _ condition: Boolean)
  */
 attribute @deny(_ operation: String, _ condition: Boolean)
 
+/**
+ * Defines the model to use when when checking access policies.
+ */
+attribute @@auth(_ modelName: String)
```

Comment:
Shall we just declare this attribute without a parameter? My thought is to use it like the following:

```prisma
model Foo {
    ...
    @@auth()
}
```

Reply:
Ah, so you were thinking the attribute would declare what Model is the "User" model. Ok, for some reason, I thought you wanted it to be declared in each place it was used, but that makes more sense.